README.md

```diff
@@ -275,7 +275,7 @@ NOTE: If you're behind a proxy, use `sudo -E` to preserve user environment.
 
 ```bash
 cd tdx/attestation
-sudo ./setup-attestation-host.sh
+sudo ./setup-host.sh
```

Comment:
Although the script is "setup-host.sh", I suggest changing it to "setup-attestation-host.sh" to be more explicit.  What do you think?

Reply:
Agreed - this is now updated in my last commit.